build/build.py

```diff
@@ -337,6 +337,8 @@ def write_bazelrc(*, python_bin_path, remote_build,
       if use_clang:
         f.write("build --config=nvcc_clang\n")
         f.write(f"build --action_env=CLANG_CUDA_COMPILER_PATH={clang_path}\n")
+      if enable_mosaic_gpu:
+        f.write("build --config=mosaic_gpu")
```

Comment:
@apaszke It needs a tailing `\n` when multiple configs are passed on.  For example, when passing
```
python build/build.py --enable_cuda --enable_mosaic_gpu --build_cuda_pjrt_plugin --build_gpu_plugin
```
it generates incorrect `.jax_configure.bazelrc` like `build --config=mosaic_gpubuild --config=cuda_plugin`.